tests/cli/create/create_tests.py

```diff
@@ -76,9 +80,113 @@ def test_009_create_project_with_appid(self):
         Tns.create(app_name=Settings.AppName.DEFAULT, template=Template.HELLO_WORLD_JS.local_package,
                    app_data=Apps.HELLO_WORLD_JS, update=False, app_id='org.nativescript.MyApp')
 
+    def test_010_create_project_with_named_app(self):
```

Comment:
If we use number conventions to when run jobs we will run this with high priority.
I'm not sure if the priority of test exactly app named `app` is high priority.